NMS-12729: OpenConfig streaming support (#3113)
OpenNMS/opennms: opennms-doc/guide-admin/src/asciidoc/text/telemetryd/protocols/openconfig.adoc

> OpenConfig Connector initiates a connection via gRPC and subscribe to one or more OpenConfig data paths (strings) and then forwards the data to queue.

Reviewer: ```suggestion
OpenConfig Connector initiates a connection via gRPC and subscribes to one or more OpenConfig data paths (strings) and then forwards the data to a queue.
```

----

====== Packages
At least one package must be present for the connector to function.

> Multiple packages can be used to customize the parameters subsets of nodes.

Reviewer: ```suggestion
Use multiple packages to customize the parameter's subsets of nodes.
```

Do the subsets of nodes belong to the parameter?

> The `filter` element is optional, if missing, all services with the given name will be considered.

Reviewer: ```suggestion
The `filter` element is optional; if missing, all services with the given name will be considered.
```

> Parameters are passed to the connector and are interpolated for node/interface & service level meta-data.

Reviewer: ```suggestion
Parameters are passed to the connector and are interpolated for node/interface and service-level metadata.
```
